Content.Server/Explosion/EntitySystems/ExplosionSystem.Processing.cs

```diff
@@ -488,9 +488,12 @@ private void ProcessEntity(
             && physics.BodyType == BodyType.Dynamic)
         {
             var pos = _transformSystem.GetWorldPosition(xform);
+            var dir = pos - epicenter.Position;
+            if (dir.IsLengthZero())
+                dir = _robustRandom.NextVector2().Normalized();
```

Comment:
```suggestion
                dir = _robustRandom.NextVector2();
```
NextVector2 already returns a normalized vector if you don't pass a magnitude.

Reply:
NextVector2 actually will generate a vector with magnitude between 0 and 1 (excluding), according to comment and code.

Reply:
nevermind, you are right